### PR TITLE
fix link in docs

### PR DESCRIPTION
**Pull request by lukaspie**

### CODE REVIEW

**Comment by rettigl** [comment]
There is also a deprecated link here (however not in docs): https://github.com/FAIRmat-NFDI/pynxtools/blob/bda1ed33e637c3616ad4f899fb3831f04924fbe5/examples/mpes/README.md?plain=1#L33

**Comment by lukaspie** [comment]
> There is also a deprecated link here (however not in docs):
> 
> https://github.com/FAIRmat-NFDI/pynxtools/blob/bda1ed33e637c3616ad4f899fb3831f04924fbe5/examples/mpes/README.md?plain=1#L33

I was anyway wondering if it makes sense to remove these READMEs for the individual plugin or at least move the content to the plugin repos. What do you think?

**Comment by rettigl** [comment]
Probably makes sense. There are also links in the docs to the plugins. Then these could rather point to the readme, than just the repo.

**Comment by lukaspie** [comment]
> Probably makes sense. There are also links in the docs to the plugins. Then these could rather point to the readme, than just the repo.

Let's remove it then. We can also start setting up rudimentary docs for `pynxtools-mpes` and add the content there.

File: examples/README.md [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
-using the [dataconverter](../pynxtools/dataconverter) tool.
-This tool offers parsers/readers/data extractors for various experimental techniques via
-technique specific plugins.
+using the [built-in readers of pynxtools](https://fairmat-nfdi.github.io/pynxtools/reference/built-in-readers.html).
 
-The examples contain code snippets for creating a NeXus/HDF5 file for the experimental technique
-according to a standardized NeXus application definition (e.g. NXem, NXmpes, NXellipsometry,
-NXapm, NXopt, NXxps, NXraman).
-Respective [Jupyter Notebooks](https://jupyter.org/) are used for running these examples.
+There is also [documentation](https://fairmat-nfdi.github.io/pynxtools/learn/dataconverter-and-readers.html) of the [dataconverter](../pynxtools/dataconverter) available. You can write a [reader](../pynxtools/dataconverter/readers) for your experimental technique
```
[the remark this answers is not in the document]
Comment (rettigl) [reply examples/README.md:6]: This link is also broken.

[the remark this answers is not in the document]
Comment (rettigl) [reply examples/README.md:6]: As well as the dataconverter link

**Review by rettigl — APPROVED** [review]
Some links need fixing, otherwise lgtm

File: examples/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukaspie) [reply examples/README.md:6]: removed links to reader, fixed links to dataconverter

**Pull request merged**